sdk/storage/blob/2019-07-07/azblob/examples_test.go

```diff
@@ -104,38 +135,14 @@ func ExampleBlockBlobOperations_Upload() {
 	// Output: 201
 }
 
-/*func ExampleContainerOperations_ListBlobFlatSegment() {
-	endpoint := pathJoin(storageEndpoint, containerName)
-	client, err := NewClient(endpoint, getCredential(), nil)
-	if err != nil {
-		panic(err)
-	}
-	blobClient := client.ContainerOperations()
-	page, err := blobClient.ListBlobFlatSegment(nil)
-	if err != nil {
-		panic(err)
-	}
-	for page.NextPage(context.Background()) {
-		resp := page.PageResponse()
-		fmt.Println(*resp.EnumerationResults)
-	}
-	if err = page.Err(); err != nil {
-		panic(err)
-	}
-	if page.PageResponse() == nil {
-		panic("unexpected nil payload")
-	}
-}*/
-
 func ExampleAppendBlobOperations_Create() {
 	endpoint := pathJoin(getEndpoint(), containerName, appendBlobName)
 	client, err := NewClient(endpoint, getCredential(), nil)
 	if err != nil {
 		panic(err)
 	}
 	blobClient := client.AppendBlobOperations()
-	s := "text/plain"
-	a, err := blobClient.Create(context.Background(), int64(10), &AppendBlobCreateOptions{BlobContentType: &s})
+	a, err := blobClient.Create(context.Background(), 0, nil)
```

Comment:
I think we can close the issue related to content type being required in the autorest.go repo due to this. Strange behavior that it worked previously, but clearly it's not required.